apps/web/components/eventtype/EventTypeSingleLayout.tsx

```diff
@@ -133,6 +134,60 @@ function getNavigation({
   ];
 }
 
+function DeleteDialog({
+  isManagedEvent,
```

Comment:
on this page https://github.com/calcom/cal.com/blob/main/apps/web/components/eventtype/EventTypeSingleLayout.tsx#L311 and https://github.com/calcom/cal.com/blob/main/apps/web/components/eventtype/EventTypeSingleLayout.tsx#L427

Reply:
Missing shouldDirty or something else? 🤔

Reply:
yeah